VocaDb.ResXFileCodeGenerator/FileOptions.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace VocaDb.ResXFileCodeGenerator;
+
+public sealed record FileOptions
+{
+	public string InnerClassInstanceName { get; init; }
+	public string InnerClassName { get; init; }
+	public InnerClassVisibility InnerClassVisibility { get; init; }
+	public bool PartialClass { get; init; }
+	public bool StaticMembers { get; init; } = true;
+	public AdditionalText File { get; init; }
```

Comment:
Isn't it possible to make the `File` property nullable rather than assigning `null!` to this property in the constructor?
I think `file.File.GetText(ctx.CancellationToken)` can be replaced with `file.File?.GetText(ctx.CancellationToken)` in the `SourceGenerator` class.

Reply:
Ofcourse it is. However, it is only ever null in testing, thus my opinion is that it should self document that this cannot ever be null in real code.